package.json

```diff
@@ -18,6 +18,9 @@
     "build/index.js",
     "build/example.js"
   ],
+  "dependencies": {
+      "prop-types": "^15.5.10"
```

Comment:
Are we ready for this? Calypso isn't on React 15.5 yet, I would recommend moving this to a peer dependency, and bump the react peer dependency to >=15.5.

The package version should probably be bumped to `"version": "2.0.0-alpha.1"`